src/main/java/com/minecolonies/client/gui/WindowHutGuardTower.java

```diff
@@ -0,0 +1,467 @@
+package com.minecolonies.client.gui;
+
+import com.blockout.Pane;
+import com.blockout.controls.Button;
+import com.blockout.controls.Label;
+import com.blockout.views.ScrollingList;
+import com.blockout.views.SwitchView;
+import com.minecolonies.MineColonies;
+import com.minecolonies.colony.buildings.BuildingGuardTower;
+import com.minecolonies.lib.Constants;
+import com.minecolonies.network.messages.GuardScepterMessage;
+import com.minecolonies.network.messages.GuardTaskMessage;
+import com.minecolonies.util.LanguageHandler;
+import net.minecraft.client.entity.EntityPlayerSP;
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+
+/**
+ * Window for the guardTower hut
+ */
+public class WindowHutGuardTower extends AbstractWindowWorkerBuilding<BuildingGuardTower.View>
+{
+    /**
+     * Id of the list of the patrol points in the GUI.
+     */
+    private static final String LIST_LEVELS    = "positions";
+
+    /**
+     * Id of the actions page in the GUI.
+     */
+    private static final String PAGE_ACTIONS    = "levelActions";
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)